app/dashboard/management/commands/activity_report.py

```diff
@@ -0,0 +1,133 @@
+            'amount_usdt': bounty.value_in_usdt,
+            'from_address': bounty.bounty_owner_address,
+            'claimee_address': bounty.claimeee_address,
+            'repo': self.extract_github_repo(bounty.github_url),
+            'from_username': bounty.bounty_owner_github_username or '',
+            'claimee_github_username': bounty.claimee_github_username or '',
+            'status': bounty.status,
+            'comments': ''
+        }
+
+    def format_tip(self, tip):
+        return {
+            'type': 'tip',
+            'created_on': tip.created_on,
+            'last_activity': tip.modified_on,
+            'amount': tip.amount,
+            'denomination': tip.tokenName,
+            'amount_eth': tip.value_in_eth,
+            'amount_usdt': tip.value_in_usdt,
+            'from_address': '',
+            'claimee_address': '',
+            'repo': self.extract_github_repo(tip.github_url) if tip.github_url else '',
+            'from_username': tip.from_email, # user tipper's email for now so tips are somewhat identifiable
+            'claimee_github_username': '', # don't know who recieves this tip at the moment
+            'status': tip.status,
+            'comments': ''
+        }
+
+
+    def handle(self, *args, **options):
+        bounties = Bounty.objects.filter(
+            network='mainnet',
+            current_bounty=True,
+            web3_created__gte=options['start_date'],
+            web3_created__lte=options['end_date']
+        ).order_by('web3_created', 'id')
+
+        tips = Tip.objects.filter(
+            network='mainnet',
+            created_on__gte=options['start_date'],
+            created_on__lte=options['end_date']
+        ).order_by('created_on', 'id')
+
+        formatted_bounties = imap(self.format_bounty, bounties)
+        formatted_tips = imap(self.format_tip, tips)
+
+        csvfile = StringIO.StringIO()
+        csvwriter = csv.DictWriter(csvfile, fieldnames=[
+            'type', 'created_on', 'last_activity', 'amount', 'denomination', 'amount_eth',
+            'amount_usdt', 'from_address', 'claimee_address', 'repo', 'from_username',
+            'claimee_github_username', 'status', 'comments'])
+
+        has_rows = False
+        for item in itermerge(formatted_bounties, formatted_tips, lambda x: x['created_on']):
+            has_rows = True
+            csvwriter.writerow(item)
+
+        start = options['start_date'].strftime(DATE_FORMAT)
+        end = options['end_date'].strftime(DATE_FORMAT)
+        if has_rows:
+            subject = 'Gitcoin Activity report from %s to %s' % (start, end)
+            send_mail(settings.CONTACT_EMAIL, settings.CONTACT_EMAIL, subject, csvfile.getvalue())
+
+            self.stdout.write(self.style.SUCCESS('Sent activity report from %s to %s to %s' % (start, end, settings.CONTACT_EMAIL)))
+        else:
+            self.stdout.write(self.style.WARNING('No activity from %s to %s to report' % (start, end)))
```

Comment:
TODO: actually send the email

Reply:
im using `marketing.mails.send_mail` on L129

* i'm assuming this method of emailing is fine, if not let me know what you prefer
* did you want an email to be sent no matter what?

Reply:
> i'm assuming this method of emailing is fine, if not let me know what you prefer

yes

> did you want an email to be sent no matter what?
yes